jetty-servlet/src/main/java/org/eclipse/jetty/servlet/ServletContextHandler.java

```diff
@@ -1071,6 +1071,9 @@ private void checkDynamicName(String name)
         {
```

Comment:
perhaps rename this method to just checkDynamic

Reply:
Done.